testsuite/integration/smoke/src/test/java/org/jboss/as/test/smoke/webservices/WSTestCase.java

```diff
@@ -98,16 +107,17 @@ public void testManagementDescription() throws Exception {
         operation.get(ModelDescriptionConstants.RECURSIVE).set(true);
 
         final ModelNode result = managementClient.getControllerClient().execute(operation);
-        Assert.assertEquals(ModelDescriptionConstants.SUCCESS, result.get(ModelDescriptionConstants.OUTCOME).asString());
-
+        assertThat(result.get(ModelDescriptionConstants.OUTCOME).asString(), is(ModelDescriptionConstants.SUCCESS));
```

Comment:
Well it helps has you won't get a null assertion for the assertTrue but a more meaningfull message
As for assertEquals I changed those where we are testing for success on operation result